src/main/java/Command.java

```diff
@@ -0,0 +1,7 @@
+public interface Command {
```

Comment:
I think it'll be great if javadocs can be added to summarise the interface's functionalities?